test/printer_test.rb

```diff
@@ -250,4 +267,139 @@ def ruby_png_fixture
     @ruby_png_fixture ||= File.open(fixture_file('ruby.png'))
   end
 
+  def real_jobs_hash
+    {
+      "success" => true,
+      "request" => {
+        "time" => "0",
+        "params" => {
+          "owner" => [
+            ""
+          ],
+          "q" => [
+            ""
+          ],
+          "offset" => [
+            ""
+          ],
+          "limit" => [
+            ""
+          ],
+          "printerid" => [
+            "c3066017-5787-d8ac-1611-cc7a3e05cff0"
+          ],
+          "sortorder" => [
+            ""
+          ],
+          "status" => [
+            ""
+          ]
+        },
+        "user" => "donald@trump.com",
```

Comment:
I'd rather we went with something else for this sample data, if you don't mind -- I'd just rather we keep politics and code apart if that's OK.